editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -764,13 +764,13 @@ const char *EditorAssetLibrary::sort_text[SORT_MAX] = {
 };
 
 const char *EditorAssetLibrary::support_key[SUPPORT_MAX] = {
-	"official",
+	"official", // Former name for the Featured support level (still used on the API backend).
```

Comment:
I like Feature so much more! Is there a path for "official" here to be replaced eventually? Do we need to update the website to serve both?

Reply:
We could figure out a way to have the API respond to the `featured` support level as if it was `official` (while keeping the response for `official` as it is). Support levels are sent as strings to the URL query string.

However, I don't have a local development setup of the asset library to test this development currently.

Reply:
@Gramps is this something you could look into?

Reply:
I have a copy setup locally but could never get the login working; granted I didn't spend much time trying.  But yeah, I can dig into the code and see what needs changed.